shoop_tests/core/test_contacts.py

```diff
@@ -73,9 +74,8 @@ def test_anonymous_contact():
         a1.delete()
 
     assert isinstance(a1.groups, QuerySet)
-    assert a1.groups.count() == 0
-    assert len(a1.groups) == 0
-    assert not a1.groups
+    assert a1.groups.count() == 1
```

Comment:
This should probably assert that the single group returned is what you expect instead.